Release.toml

```
@@ -311,5 +311,6 @@ version = "1.21.0"
     "migrate_v1.20.0_public-admin-container-v0-11-8.lz4",
     "migrate_v1.20.0_aws-control-container-v0-7-12.lz4",
     "migrate_v1.20.0_public-control-container-v0-7-12.lz4",
+    "migrate_v1.20.0_k8s-reserved-cpus-v0-1-0.lz4",
```

You'll want to put this in the next release. The list is currently empty but this would be the first one.

```suggestion
]
 "(1.20.0, 1.21.0)" = [
       "migrate_v1.21.0_k8s-reserved-cpus-v0-1-0.lz4",
]
```

---

This should be fixed now.